http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java

```diff
@@ -33,6 +35,18 @@ public abstract class AbstractVertxIT {
     @QuarkusApplication
     static final RestService service = new RestService();
 
+    Vertx vertx;
+    HttpClient httpClient;
```

Comment:
Vert.X and HttpClients are created for each single unit test,therefore there is no point really to storing them in class member variable. you can use try-with-resources inside method annotated with @Test . The reason why I suggested to assign Vert.X to the static var was that you could create it one in @BeforeAll. However I'm fine with creating it every single time, in which case please drop all class member vars and use try-with-resources